deploy/kubernetes/base/db-deployment.yaml

```diff
@@ -1,10 +1,6 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  annotations:
-    kompose.cmd: kompose convert --file ../../docker-compose.yml
```

Comment:
I'm guessing this was added as a hint to how the .yaml file was created? If that's the case, it seems mildly useful to keep (if possibly outdated, from the looks of the path) Maybe add this information as part of ./deploy/kubernetes/README.md if maintaining these annotations proves too cumbersome,

Just curious what the log bloating you refer to is.

Reply:
This was the original command that generated these files, but the two configs have now significantly diverged to the point that it is no longer useful to document this.

Reply:
Ah ha - so these files have now been manually changed, since their original `kompose convert` creation. Makes sense to not document in that case. One question, though, what is the relationship if any between the docker-compose file and the kube files, then? Why do we use one over the other?